FX2024 Domain & Capabilities update / Optimize / Cloud Sustainability (#683)
microsoft/finops-toolkit: docs/_docs/framework/capabilities/optimize/sustainability.md

---
layout: default
grand_parent: FinOps Framework
parent: Optimize
title: Cloud sustainability
permalink: /framework/capabilities/optimize/sustainability

> nav_order: 1

Reviewer: ```suggestion
nav_order: 5
```

description: This article helps you understand the cloud sustainability capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
author: bandersmsft
ms.author: banders

> ms.date: 04/09/2024

Reviewer: ```suggestion
ms.date: 06/12/2024
```

ms.topic: conceptual
ms.service: finops
ms.reviewer: micflan
---

<span class="fs-9 d-block mb-4">Cloud sustainability</span>
This article helps you understand the cloud sustainability capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
{: .fs-6 .fw-300 }

<details open markdown="1">
  <summary class="fs-2 text-uppercase">On this page</summary>

- [▶️ Getting started](#️-getting-started)
- [🏗️ Building on the basics](#️-building-on-the-basics)
- [🍎 Learn more at the FinOps Foundation](#-learn-more-at-the-finops-foundation)
- [⏩ Next steps](#-next-steps)
- [🧰 Related tools](#-related-tools)

</details>

---

<a name="definition"></a>

> **Cloud sustainability defines how organization's decision-making process regarding cloud usage taking into account its environmental impact and aligning with broader sustainability objectives."**

Reviewer: ```suggestion
**Cloud sustainability balances environmental and financial efficiency in cloud optimization, ensuring alignment with strategic objectives.**
```

-Review the [Microsoft Cloud for Sustainability design principles](https://learn.microsoft.com/industry/well-architected/sustainability/cloud-sustainability-design-principles) outlined as part of the Well-Architected for Sustainability guidance.
- Use the [Azure carbon optimization](https://learn.microsoft.com/azure/carbon-optimization/overview) to find opportunities to optimize your Azure environment to lower your carbon emissions and costs.
- Calculate your cloud-based carbon emissions with the [Emissions Impact Dashboard for Azure](https://learn.microsoft.com/power-bi/connect-data/service-connect-to-emissions-impact-dashboard) or the [Emissions Impact Dashboard for Microsoft 365](https://learn.microsoft.com/power-bi/connect-data/service-connect-to-emissions-impact-dashboard).
- Review the Microsoft Cloud for Sustainability [reference architectures](https://learn.microsoft.com/industry/well-architected/sustainability/sustainability-architecture-overview) to identify which tools are more relevant for your environmental, social, and governance (ESG) capabilities.

> <br>

Reviewer 2: Should we also highlight the Sustainability section of the Advisor Cost Optimization workbook?

## 🏗️ Building on the basics

> -Deploy the [Sustainability data solutions in Microsoft Fabric ](https://learn.microsoft.com/industry/sustainability/sustainability-data-solutions-overview) to help you ingest, harmonize, and process data for specific sustainability scenarios, such as ESG data estate, Microsoft Azure emissions insights, and Social and governance metrics and reports.
> 
> NOTE:Some or all of this functionality is available as part of a preview release. The content and the functionality are subject to change.

Reviewer: ```suggestion
At this point, you've implemented all the basic carbon optimization recommendations and tuned applications to meet the most fundamental best practices. As you move beyond the basics, consider the following points:

- Expand visibility to business and finance stakeholders by using the [Emissions Impact Dashboard for Azure](https://learn.microsoft.com/power-bi/connect-data/service-connect-to-emissions-impact-dashboard) or the [Emissions Impact Dashboard for Microsoft 365](https://learn.microsoft.com/power-bi/connect-data/service-connect-to-emissions-impact-dashboard).
- Explore additional tools available in Microsoft's environmental, social, and governance (ESG) capabilities available from the [Microsoft Cloud for Sustainability](https://learn.microsoft.com/industry/sustainability/overview).
  - For a more prescriptive approach, refer to the [Well-Architected guide for sustainability](https://learn.microsoft.com/industry/well-architected/sustainability).
```

<br>

## 🍎 Learn more at the FinOps Foundation

This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Cloud sustainability](https://www.finops.org/framework/capabilities/cloud-sustainability/) article in the FinOps Framework documentation.

> You can also find related videos on the FinOps Foundation YouTube channel:
> 
> <!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->
> {% include video.html title="Resource utilization and efficiency videos" id="DIcO8EulN8PuXuWL" list="PLUSCToibAswlL6Ms76cl9GDmcpM85nlWA" %}

Reviewer: ```suggestion
```
